### ISSUE TITLE
CardFormField's CardFormStyle not working

### ISSUE THREAD

**Issue by ShwetaChauhan18** [flutter-stripe/flutter_stripe#731]
**CardFormField new style is not working. Only backgroundColor is changing.**

```
Flutter 3.0.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision ee4e09cce0 (7 days ago) • 2022-05-09 16:45:18 -0700
Engine • revision d1b9a6938a
Tools • Dart 2.17.0 • DevTools 2.12.2
shwetachauhan@SF-LAP-160 flutter_stripe % 
```

```
style: CardFormStyle(
     textColor: AppColors.textColor,
     fontSize: 18,
     backgroundColor: Colors.white,
     borderColor: AppColors.lightPink,
     cursorColor: AppColors.textColor,
     placeholderColor: AppColors.textColor,
     textErrorColor: AppColors.textColor,
 ),
```

**Comment by remonh87** [comment]
can you tell me for which platform?

**Comment by ShwetaChauhan18** [comment]
In both platform

**Comment by ShwetaChauhan18** [comment]
<img width="409" alt="image" src="https://user-images.githubusercontent.com/34509457/168780435-3f77179b-a7dd-4bc0-a119-ca2bd10212e1.png">

<img width="328" alt="image" src="https://user-images.githubusercontent.com/34509457/168780683-36cc1353-1715-4e6d-8d1d-a856c90ae360.png">

**Comment by ShwetaChauhan18** [comment]
Also, after confirm payment it's open web hook. Can we restrict that?

**Comment by korzonkiee** [comment]
The CardFormStyle doesn't work for me on Android (Xiaomi Mi A2 Lite, Android 9). I'm on the latest flutter_stripe: 3.0.0 version.

https://user-images.githubusercontent.com/26259846/168812045-e84a0dd4-7967-455f-bbf4-4a312660fbe5.mp4

### Environment

flutter_stripe 

<details>
  <summary markdown="span">flutter --version</summary>
Flutter 3.0.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision ee4e09cce0 (8 days ago) • 2022-05-09 16:45:18 -0700
Engine • revision d1b9a6938a
Tools • Dart 2.17.0 • DevTools 2.12.2
</details>

<details>
  <summary markdown="span">flutter doctor</summary>
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.0.0, on macOS 12.3.1 21E258 darwin-arm, locale en-PL)
[!] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.
[✓] Xcode - develop for iOS and macOS (Xcode 13.3.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.67.1)
[✓] Connected device (6 available)
[✓] HTTP Host Availability

! Doctor found issues in 1 category.

</details>

**Comment by korzonkiee** [comment]
Actually, it doesn't seem to work on iOS as well. I changed the textColor property to red and it still displays in black.

```dart
class _CardField extends StatelessWidget {
  const _CardField({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return CardFormField(
      style: CardFormStyle(
        fontSize: 24,
        textColor: AppColors.red,
        cursorColor: AppColors.black,
        borderColor: AppColors.black,
        textErrorColor: AppColors.red,
        placeholderColor: AppColors.dividerGrey,
      ),
      autofocus: true,
      controller: context.read<CardFormEditController>(),
    );
  }
}
```

![IMG_7DF67FA1EC0F-1](https://user-images.githubusercontent.com/26259846/168816014-4a985dad-4758-4b8c-b08d-147bd0594a48.jpeg)

**Comment by remonh87** [comment]
#749 will fix the styling on Android but on iOS we need to wait on the [Stripe sdk](https://github.com/stripe/stripe-react-native/issues/563)

**Comment by priyankabambharoliya-official** [comment]
I have same issue

**Comment by remonh87** [comment]
Fixed in version 3.1.0

**Comment by korzonkiee** [comment]
I keep having very strange behavior on Android. I'm using flutter_stripe 3.1.0. Tested both on debug & release.

https://user-images.githubusercontent.com/26259846/173819024-2901befa-18dc-4676-9647-8b4f693db15f.mp4
